### DISCUSSION TITLE
No package metadata was found for ckan

### DISCUSSION THREAD

**Discussion by Oswaldoivann** [ckan/ckan#7532] [Q&A]
I have installed Ubuntu 20.04 on my PC. Following the CKAN 2.10 installation steps, I have completed the initial steps but when trying to install the packages with "sudo dpkg -i", it throws an error: "Error: importlib.metadata.PackageNotFoundError: No package metadata was found for ckan."

I have created a document for installing the software and you can see the commands used in the previous steps.

https://docs.google.com/document/d/16s7QrQh8ka-PtTIYsxkvxKdMQc0shUfZJ6A2BL3KDnY/edit?usp=sharing

**Comment by muhammetcetinyamac** [comment]
Can you try again by installing package importlib-metadata?

`pip install importlib-metadata`
